packages/monaco/src/browser/monaco-diff-editor.ts

```diff
@@ -40,21 +44,35 @@ export class MonacoDiffEditor extends MonacoEditor {
         const original = originalModel.textEditorModel;
         const modified = modifiedModel.textEditorModel;
         this._diffEditor.setModel({ original, modified });
+        this.updateDiffNavigator(options || {});
     }
 
     get diffEditor(): IStandaloneDiffEditor {
         return this._diffEditor;
     }
 
+    get diffNavigator(): IDiffNavigator {
```

Comment:
Is there a plan to export it `@theia/editor` or we are going to use it directly in `@theia/git`?

Also looking at monaco API, it does not seem that public API allows checking whether the selected change is last or next.

Reply:
@akosyakov that's really bad. reading sources of `DiffNavigator` shows, that's all privates. it would be so easy to check `hasNext`/`hasPrevious`, if ranges and index would be accessible. would copying of https://github.com/Microsoft/vscode/blob/master/src/vs/editor/browser/widget/diffNavigator.ts be an option?

Reply:
Should we add and customize the DiffNavigator implementation?

Reply:
I've updated the branch, introducing `DiffNavigator` in `@theia/editor`.